domains/transfer/Transfer.API/API/Shared/Data/ApplicationDbContext.cs

```diff
@@ -25,5 +28,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<TransferAgreement>()
             .HasIndex(nameof(TransferAgreement.SenderId), nameof(TransferAgreement.TransferAgreementNumber))
             .IsUnique();
+
+        modelBuilder.Entity<ClaimSubject>()
+            .HasKey(p => p.SubjectId);
+
+        modelBuilder.Entity<ClaimSubjectHistory>()
+            .HasKey(p => p.SubjectId);
```

Comment:
This defines the primary key to be "SubjectId", but that will mean, that it is not possible to have multiple entries for the same subject id. Is this a mistake?

Reply:
No since you can only start the robot once pr subjectId

Reply:
It does not make sense to set the primary key for ClaimSubjectHistory to SubjectId, when there is also an Id property.

Reply:
Sorry - thought we were talking about ClaimSubject. Agreed, this does not make sense. But this is to be removed so doesn't matter anymore